packages/react-components/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -11,39 +11,40 @@ const useStyles = makeStyles({
     display: 'flex',
     flexWrap: 'wrap',
     justifyContent: ' flex-start',
+    backgroundColor: '#F7F7F7',
 
-    '> span': {
-      alignItems: 'center',
-      color: '#3b3a39',
-      display: 'flex',
-      flexDirection: 'column',
-      height: '80px',
-      justifyContent: 'space-around',
-      ...shorthands.padding(0),
-      width: '80px',
-      ...shorthands.overflow('hidden'),
+    '> div': {
```

Comment:
is there a reason why you need this div? If you use grid you can use gap instead and probably remove this div. Since you are rendering many icons, it could help with perf.